source/uwp/AdaptiveCardsObjectModel/idl/AdaptiveCards.ObjectModel.Uwp.idl

```diff
@@ -324,6 +324,14 @@ namespace AdaptiveCards.ObjectModel.Uwp
         Windows.Foundation.Collections.IVector<IAdaptiveInline> Inlines { get; };
     };
 
+    runtimeclass AdaptiveCaptionSource
```

Comment:
This is exactly the same as `AdaptiveMediaSource` - let's refactor into a single base class with two children?:)

Reply:
Yep, done